app/services/conference_domains_service.rb

```diff
@@ -0,0 +1,27 @@
+class ConferenceDomainsService
```

Comment:
Oh yes, I forgot to mention that to add tests we will need a couple of domains in which one has a CNAME record for another domain.
  
I can use `osem-demo.herokuapp.com` and `demo.osem.io` for now but the tests will fail if we ever switch the staging app to another domain.